### PR TITLE
[EOSF-457] Feature/yarn

### PR DESCRIPTION
**Pull request by binoculars**
## Ticket

https://openscience.atlassian.net/browse/EOSF-457

## Purpose

Updates the node package installation process to use yarn instead
Update ember-cli for the project

## Changes

Install yarn https://yarnpkg.com/

To get tests passing locally, you may have to do the setup steps for ember-cli in https://github.com/ember-cli/ember-cli/releases/tag/v2.10.0

### CODE REVIEW

File: Dockerfile
Comment (abought) [inline Dockerfile:1]: As with the similar PR for [preprints](https://github.com/CenterForOpenScience/ember-preprints/pull/249/files) : changes should be approved by DevOps.

File: README.md
Comment (abought) [inline README.md:1]: Thank you for fixing these copy-paste errors. :)

**Review by abought — APPROVED** [review]
Changes are very similar to preprints PR. Looks good to me, but Dockerfile should be approved by DevOps.

**Review by abought — CHANGES_REQUESTED** [review]
Changes look good pending DevOps approval.

However, CI is failing. Although develop branch is also failing, would it be within scope to request fixing failing tests before merge?

**Comment by binoculars** [comment]
It's out of scope for this PR. @hmoco was working on it.

File: Dockerfile
Comment (binoculars) [reply Dockerfile:1]: @mattclark this is the same Dockerfile I updated with ember-preprints. node:boron is the 6.X LTS release.

Comment (abought) [reply Dockerfile:1]: Worth noting that Michael may have already read / written the code in the related PR.

(just trying to keep info in sync across repos!)

**Comment by abought** [comment]
> It's out of scope for this PR. @hmoco was working on it.

I wasn't able to find a ticket, but in that case I'll leave final decision to @brianjgeiger (who will have better visibility on progress of these failing tests). I'll assign the JIRA ticket to him accordingly.

Because this ticket makes changes to Travis configuration, I'd feel a little more comfortable if we had a clear travis run using this configuration... But the consequences of npm are also starting to mount. Judgment call. :)

**Pull request merged**
